src/sphinx.md

```diff
@@ -57,6 +59,7 @@ These are derived as follows:
 ## MAC verification
 
 `mac` should equal the BLAKE2b hash of `actions` computed with the key `mac_key`.
+// TODO is it the first 16 byte?
```

Comment:
Not sure what you mean sorry? `mac` and `actions` are as defined in the table at the top, `mac_key` is derived as in the "Secret derivation" section.

Reply:
mac is only 16 byte long, but blake2b output 32bytes.

Reply:
BLAKE2b actually outputs 64 bytes. The MAC key (16 bytes), actions encryption key (32 bytes), and delay seed (16 bytes) are generated from a single hash:

    mac_key . actions_encryption_key . delay_seed =
        blake2b("sphinx-small-d-s", 0, kx_shared_secret)

Reply:
Would make sense to indicate it is only the first 16 bytes just to lift any misunderstanding.